scripts/mail-alarm

```diff
@@ -237,28 +238,29 @@ class NetworkUsageAlarmETG(EmailTextGenerator):
         pool_name = pool_name = get_pool_name()
```

Comment:
`pool_name = pool_name = get_pool_name()` should have been `pool_name = get_pool_name()`